service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -146,86 +117,62 @@ private void getTwinOperation(URL url, DeviceTwinDevice device) throws IotHubExc
 
     /**
      * This method updates device twin for the specified device.
+     * <p>This API uses the IoT Hub PATCH API when sending updates, but it sends the full twin with each patch update.
+     * As a result, devices subscribed to twin will receive notifications that each property is changed when this API is
+     * called, even if only some of the properties were changed.</p>
+     * <p>See <a href="https://docs.microsoft.com/en-us/rest/api/iothub/service/twin/updatedevicetwin">PATCH</a> for
```

Comment:
The link does not work (404).

Reply:
I didn't add the URL, this section was just reformatted. I concur it is an out of date URL.

Do you know the real one to use? If we can't find one, should we just remove this?

Reply:
I found this one: https://docs.microsoft.com/en-us/rest/api/iothub/service/devices/updatetwin